install/bootstrap_livecd_usb.md

```diff
@@ -355,25 +358,22 @@ Some files are needed for generating the configuration payload. See these topics
       linux# export SYSTEM_NAME=eniac
       ```
 
-   1. Generate the system configuration. See below for an explanation of the command line parameters and some common settings.
-
+   1. Generate the system config:
+      > **`NOTE`** the provided command below is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary sifnificatnly depending on ones system and site configuration.
```

Comment:
re: my previous comments about missing `csi config init` options... could maybe just replicate this disclaimer in those places too.